parsec/backend/memory/realm.py

```diff
@@ -102,6 +107,20 @@ async def get_status(
             raise RealmAccessError()
         return realm.status
 
+    async def get_stats(
+        self, organization_id: OrganizationID, author: DeviceID, realm_id: UUID
+    ) -> RealmStats:
+        from pprint import pprint
+        RealmStats.data_size = 0
+        for (_, _), blockmeta_value in self._block_component._blockmetas.items():
+            if blockmeta_value.realm_id == realm_id:
+                RealmStats.data_size += blockmeta_value.size
+        for (_, _), vlobmeta_value in self._vlob_component._vlobs.items():
```

Comment:
I think we shouldn't mix metadata & data size: it's much better to provide two separate `metadata_size` & `data_size` (so client can decide if he wants to display those data separately or together)

besides, data size is much bigger than metadata size (~500 time bigger in our production server) so adding metadata size to data size won't change much.